pkg/cmd/cli/describe/printer.go

```diff
@@ -152,7 +152,7 @@ func PrintTemplateParameters(params []templateapi.Parameter, output io.Writer) e
 func printTemplate(t *templateapi.Template, w io.Writer) error {
 	description := ""
 	if t.Annotations != nil {
-		description = t.Annotations["description"]
+		description = t.Annotations["openshift.io/description"]
```

Comment:
Template will follow a different namespace convention? #2169 references lots of annotations for Template that need updating, what to do with those?